website/docs/contract-test-collection.md

```diff
@@ -368,50 +830,80 @@ Tzip-7 introduced the approvable ledger: [Tzip-7](https://gitlab.com/tezos/tzip/
 
 - [A Beginner's Guide to Tezos Tzip-7 Prooposal](https://claudebarde.medium.com/a-beginners-guide-to-tezos-tzip-7-proposal-90a8b816af7e)
 
-This contract has an FA1.2 interface.  To determine if a contract has an FA1.2 interface we can use 
-```
-tezos-client check contract KT1CfFBaLoUrgv93k8668KCCcu2hNDNYPz4L implements fa1.2
-```
+This contract has an FA1.2 interface. 
 
 #### Entrypoints:
-  * approve
-  * burn
-  * getAdministrator
-  * getAllowance
-  * getBalance
-  * getTotalSupply
-  * mint
-  * setAdministrator
-  * setPause
-  * transfer
-
-#### Storage:
+
+- approve
+- burn
+- getAdministrator
+- getAllowance
+- getBalance
+- getTotalSupply
+- mint
+- setAdministrator
+- setPause
+- transfer
+
+<Tabs
+defaultValue="michelson"
+values={[
+{label: 'Michelson Storage', value: 'michelson'},
+{label: 'Taquito Storage', value: 'taquito'}
+]}>
+<TabItem value="michelson">
+
 ```js
 storage (pair (big_map address (pair nat (map address nat)))
               (pair address (pair bool nat)));
 ```
 
+</TabItem>
+  <TabItem value="taquito">
+
+```js
+    const op = await tezos.contract.originate({
+      balance: '1',
+      code: tokenCode,
+      init: tokenInit(await tezos.signer.publicKeyHash()),
```

Comment:
should we use `storage` instead of `init`?
`tokenInit` does not really indicate what the storage is

Reply:
I changed it to explicitly show the storage that is imported, but I'm not sure it's the best solution.